code/game/objects/structures/cannon_shell.dm

```diff
@@ -37,7 +37,7 @@
 
 /obj/item/weapon/twohanded/required/shell_casing
 	name = "cannon shell casing"
-	desc = "The ejected casing from a cannon shell. Not very useful."
+	desc = "The ejected thrust segement of the shell used to give it that extra kick"
```

Comment:
I preferred the old description, since the fluff people came up with for the MAC was kinda shitty anyways. At the very least puncutate please.